README.md

```diff
@@ -93,9 +98,15 @@ docker run -d -p 5000:5000 --env-file .env todo-app:prod
 ```
 ## DEVELOPMENT RUN
 ```bash
-docker run -d -p 5000:5000 --env-file .env \
---mount type=bind,source=$(pwd)/todo_app,target=/srv/www todo-app:dev 
+docker run -d --name developmentImage -p 5000:5000 --env-file .env --mount type=bind,source=$(pwd)/,target=/srv/www todo-app:dev
+
+Instigate tests: docker exec -it developmentImage "pytest" "--disable-pytest-warnings" "/srv/www/todo_app/tests"
 
 For docker compose, please use: docker-compose -p DevOps up --detach
 ```
 
+## TEST RUN
+```bash
+docker run -d --name TravisCIImage -p 5000:5000 --env-file .env --mount type=bind,source=$(pwd)/,target=/srv/www todo-app:test
+
+Instigate tests: docker exec -it TravisCIImage "pytest" "--disable-pytest-warnings" "/srv/www/todo_app/tests"
```

Comment:
Happy for you to stick with your preference, but worth asking why you prefer this approach for running tests, rather than setting the default command as your entrypoint? It makes sense to me for the command you offer for running tests within the development container, but less for this container as this is intended specifically for running the automated tests.